src/Buckle/Buckle/CodeAnalysis/Binding/Binder.cs

```diff
@@ -556,7 +556,7 @@ private struct BinderState {
 
         var symbols = scope_.LookupOverloads(name);
 
-        if (symbols == null) {
```

Comment:
I rewrote scope_.LookupOverloads so it shouldn't ever return null, instead, it returns an empty array. I am leaving the null check just in case, because the type is still nullable